### DISCUSSION TITLE
[潜水]有生之年能有第四季吗

### DISCUSSION THREAD

**Discussion by MIHLFY** [share121/inter-knot#526] [其他]
![20230925_010653](https://github.com/user-attachments/assets/8b58d420-c0c7-4003-9060-08e3bce2e251)
第三季完全没看过（哭）
![-1f35655b0b2ea593](https://github.com/user-attachments/assets/60105a9e-4db6-4786-a9db-c4da7be918ce)

**Comment by mixue4646** [comment]
素晴质量还是有保证的，第三季好像评价也不错，只要没啥情况应该会有第四季吧，只不过是等几年的问题了

**Reply by MIHLFY** [reply]
只能慢慢等了(´。＿。｀)
